code/mathematical_algorithms/src/add_polynomials/add_polynomials.cpp

```diff
@@ -5,50 +5,56 @@
 #include <stddef.h>
 using namespace std;
 
-struct term {
-  int coeff;
-  int pow;
-  term* next;
-  term(int, int);
-};
 
-term::term(int c, int p) {
-  coeff = c;
-  pow = p;
-  next = NULL;
-}
+class Term {
+  public:
+    Term(int c, int p) {
+      coeff = c;
+      pow = p;
+      next = NULL;
+    }
+    int coeff;
```

Comment:
If it's possible, could you make these variables private, have an underscore at the end, and have getters/setters if needed?